packages/bootstrap/extra/dcos_internal_utils/cli.py

```diff
@@ -62,6 +66,27 @@ def noop(b, opts):
 }
 
 
+def _apply_sysctl(setting, service):
+    try:
+        check_call(["sysctl", "-q", "-w", setting])
+    except CalledProcessError:
```

Comment:
Non failure here is intentional.

The sysctl settings of minuteman is operating system dependent. We don't want to fail on that.

```
Aug 04 00:31:39 dcos-docker-master1 bootstrap[1241]: sysctl: cannot stat /proc/sys/net/ipv4/netfilter/ip_conntrack_tcp_be_liberal: No such file or directory
Aug 04 00:31:39 dcos-docker-master1 bootstrap[1241]: [WARNING] sysctl net.ipv4.netfilter.ip_conntrack_tcp_be_liberal=1 not set for dcos-minuteman
Aug 04 00:31:39 dcos-docker-master1 bootstrap[1241]: sysctl: cannot stat /proc/sys/net/netfilter/ip_conntrack_tcp_be_liberal: No such file or directory
Aug 04 00:31:39 dcos-docker-master1 bootstrap[1241]: [WARNING] sysctl net.netfilter.ip_conntrack_tcp_be_liberal=1 not set for dcos-minuteman
Aug 04 00:31:39 dcos-docker-master1 bootstrap[1241]: [DEBUG] bootstrapping dcos-minuteman
```